packages/next/src/lib/metadata/generate/utils.ts

```diff
@@ -1,20 +1,17 @@
-function resolveArray<T>(value: T): T[] {
+function resolveArray<T>(value: T): T extends Array<any> ? T : T[] {
```

Comment:
A cleaner way would be:

```suggestion
function resolveArray<T>(value: T | T[]): T[] {
  if (Array.isArray(value)) {
    return value;
  } else {
    return [value];
  }
}
```